20240129-satellite-managed-encryption.md

```diff
@@ -0,0 +1,45 @@
+tags: []
```

Comment:
It would be nice to follow the template https://github.com/storj/design-docs/blob/main/00000000-template.md

There is a document that led to creating it https://storjlabs.atlassian.net/wiki/spaces/delivery/pages/2548137985/Design+Doc+Process+Proposal+2023

Unfortunately, that Confluence page isn't public, we should make it public, so people outside of Storj can find out why we have the template, etc.

Reply:
done